app/Models/ChargeGroup.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * @property int $id
+ * @property int $year
+ * @property string $charge_group_1
+ * @property string $charge_group_2
+ * @property string $grid_operator_gln
+ * @property string $grid_operator_name
+ * @property int $number_of_metering_points
+ * @property int $consumption_kwh
+ */
+class ChargeGroup extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['year', 'charge_group_1', 'charge_group_2', 'grid_operator_gln', 'grid_operator_name', 'number_of_metering_points','consumption_kwh'];
```

Comment:
Kan vi ikke bare sætte `protected $guarded = [];` for alle modeller istedet for ?

Reply:
Jo. Jeg mener, at Laravels begrundelse for ikke at gøre det, er at man skulle kunne lave ->fill(request()->all()) i en controller, uden at der sker ulykker ved det.
For det første har jeg aldrig gjort netop det, for det andet så har det, da jeg var mindre erfaren givet mig mange hovedbrud, hvorfor en property ikke blev opdateret, når jeg lavede Model::create([.. - i dag er jeg vant til at kigge efter, om det her har givet problemer.
Men jeg har altid troligt lagt properties i $fillable og så kørt videre, så det er nok tid til at sige, at sådan gør vi bare, og så er vi bare det bedre, når vi skriver controller-metoder :+1: